server/bundler/utils.js

```diff
@@ -14,6 +14,9 @@ function getAssets( stats ) {
 }
 
 function pathToRegExp( path ) {
+	if ( path === '/' ) {
+		return path;
+	}
 	return new RegExp( '^' + path + '(/.*)?$' );
```

Comment:
> I feel like we should be able to generate the sections route list using the sections config, and perhaps a bit more mapping, or using additional convention in each section.

Yes, as @mtias and @ockham and I discussed today, I think this needs a broader rethink of routing, and where we consider the source of truth for routes to be. e.g. if we have them in the `sections.js`, how do we specify middlewares? Is it more readable to have each section define its own routes in its `index.js`? How do we handle isomorphic routes? How do we DRY?

Ideally routing would become more declarative, in any case.